src/frontend/js/components/DashboardAddressesManagement/DashboardCreateAddressLoader.tsx

```diff
@@ -0,0 +1,11 @@
+import { DashboardCreateAddress } from 'components/DashboardAddressesManagement/DashboardCreateAddress';
+import { DashboardPaths } from 'utils/routers/dashboard';
+import { useDashboardNavigate } from 'utils/routers/dashboard/useDashboardRouter';
+
+/**
+ * This component relies on react-router.
```

Comment:
```suggestion
 * This component relies on react-router.
 * Read https://reactrouter.com/en/6.4.1/route/loader for further information.
```

Reply:
In reality it is not a loader in terms of `react-router` ( we were not able to use hooks in them ), so isn't that misleading to redirect to the loaders doc ? 🤔

Reply:
Indeed it can be confusing.